test/Microsoft.Dnx.Testing.CommonUtils/TestUtils.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Dnx.Runtime;
+using Microsoft.Dnx.Runtime.Helpers;
+using System;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Runtime.Versioning;
+
+namespace Microsoft.Dnx.Testing
+{
+    public static class TestUtils
+    {
+        public static FrameworkName GetFrameworkForRuntimeFlavor(string flavor)
+        {
+            if (string.Equals("clr", flavor, StringComparison.OrdinalIgnoreCase) ||
+                string.Equals("mono", flavor, StringComparison.OrdinalIgnoreCase))
+            {
+                return FrameworkNameHelper.ParseFrameworkName("dnx451");
+            }
+            else if (string.Equals("coreclr", flavor, StringComparison.OrdinalIgnoreCase))
+            {
+                return FrameworkNameHelper.ParseFrameworkName("dnxcore50");
+            }
+
+            throw new InvalidOperationException($"Unknown runtime flavor '{flavor}'");
+        }
+
+        public static Solution GetSolution<TTest>(
+            DnxSdk sdk, 
+            string solutionName, 
+            [CallerMemberName]string testName = null, 
```

Comment:
That's ok, we can make sure it holds in code reviews. This is the best way we know the get the test name.